packages/framework/tests/Unit/Component/Translation/Resources/Controller.php

```diff
@@ -4,24 +4,20 @@
 
 use JMS\TranslationBundle\Annotation\Ignore;
 use Shopsys\FrameworkBundle\Component\Translation\Translator;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller as BaseController;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 
-class Controller extends BaseController
+class Controller extends AbstractController
 {
     public function indexAction()
     {
         /** @var \Shopsys\FrameworkBundle\Component\Translation\Translator $translator */
         $translator = $this->get(Translator::class);
 
         $translator->trans('trans test');
-        $translator->transChoice('transChoice test', 5);
```

Comment:
should not this be rather replaced by something like `$translator->trans('trans with count test', 5);`? :thinking: 
The question applies to all the deletions in this file

Reply:
but now I see it is just the extraction test so it does not make sense to test the `trans` method multiple times, right?

Reply:
Yes, it tested nothing